dockers/docker-orchagent/swssconfig.sh

```diff
@@ -29,6 +29,25 @@ function fast_reboot {
   esac
 }
 
+function config_reload {
```

Comment:
can you rename this function as restator_neighbors, and then rename the fast_reboot script as restore default_routes. basically, i think we should try to reuse the code here and not to create duplication.

Reply:
I can refactor this to re-use more existing code, but not by much since we don't want a lot of overlapping behavior between fast-reboot and config-reload here. E.g. for fast-reboot, we want to restore the ARP table to APP_DB as well as the default routes, but for config reload we do not want to restore the default routes, and we only need the ARP table to be restored to the kernel (if we restored to APP_DB here, there was some concern about having these entries be `FAILED` if the ToR attempted an ARP refresh).